crm_timesheet/models/crm_phonecall.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# See README.rst file on addon root folder for license details
+
+from openerp import models, fields, api, _
+from openerp.fields import DATE_LENGTH
+from datetime import datetime
+from openerp.exceptions import ValidationError
+
+
+class CrmPhonecall(models.Model):
+    _inherit = 'crm.phonecall'
+
+    analytic_account_id = fields.Many2one(
+        comodel_name='account.analytic.account')
+    timesheet_ids = fields.One2many(comodel_name='hr.analytic.timesheet',
+                                    inverse_name='phonecall_id')
+
+    def _timesheet_prepare(self, vals):
+        """
+            param: vals
+            type: dict
+            desc: crm.phonecall vals dict (It's like create and write methods)
+        """
+        if len(self) > 0:
```

Comment:
This case is for create case, when ```self.ensure_one()``` raise an exception. This method accept recordsets of 0 or 1 record, but no more.